CONTRIBUTING.md

```diff
@@ -177,6 +177,30 @@ To maintain understandable code and avoid circular dependencies it is important
 - **templates:** Golang templates for generating the html output.
 - **vendor:** External code that Gitea depends on.
 
+## API v1
+
+The API is documented by [swagger](http://gitea.com/api/swagger) and is based on [GitHub API v3](https://developer.github.com/v3/).
```

Comment:
imo this should use `try.gitea.io` instead of `gitea.com`  
try will always follow master, whereas gitea.com may not